### PR TITLE
added values to animal category

### PR DESCRIPTION
**Pull request by mephi007**

### CODE REVIEW

**Comment by gabrielbarker** [comment]
Hey @mephi007 
Thanks a lot for the contribution! I've just spotted a couple of tweaks but other than that it looks great! I'll add comments

File: qual_id/categories/games.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class Games(Category):
+  def get_values(self):
+    return [
+        'basketball',
+        'cricket',
+        'baseball',
+        'hockey',
+        'chess',
+        'tennis',
+        'table-tennis',
```
Comment (gabrielbarker) [inline qual_id/categories/games.py:13]: Would you mind removing 'table-tennis' or replacing it as it contains a '-' which could be confusing. 
I meant to write tests to cover this but I forgot so thanks for reminding me 🙂

File: test/categories/test_games.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.games import Games
+import random
+
+
+class TestAnimal(unittest.TestCase):
```
Comment (gabrielbarker) [inline test/categories/test_games.py:6]:
```suggestion
class TestGame(unittest.TestCase):
```

File: qual_id/categories/games.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from ..category import Category
+
+
+class Games(Category):
```
Comment (gabrielbarker) [inline qual_id/categories/games.py:4]:
```suggestion
class Game(Category):
```

File: qual_id/pattern.py [hunk 4]
```diff
@@ -13,6 +14,7 @@ class Pattern:
       "food": Food(),
       "drink": Drink(),
       "instrument": Instrument(),
+      "games":Games()
```
Comment (gabrielbarker) [inline qual_id/pattern.py:17]:
```suggestion
      "game": Game()
```

File: test/categories/test_games.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  def setUp(self):
+    self.games = Games()
```
Comment (gabrielbarker) [inline test/categories/test_games.py:8]:
```suggestion
    self.games = Game()
```

File: test/categories/test_games.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.games import Games
```
Comment (gabrielbarker) [inline test/categories/test_games.py:2]:
```suggestion
from qual_id.categories.game import Game
```

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
I've pointed out some places where you've used `Games` instead of `Game`. The other classes follow the pattern of being _singular_. If you wouldn't mind changing these **and** changing the names of the file 'games.py' to 'game.py' and 'test_games.py' to 'test_game.py'
Thank you!

File: qual_id/categories/games.py (hunk 1 above)
Comment (mephi007) [reply qual_id/categories/games.py:13]: Hey @gabrielbarker, can't we use camelCase or use '_' instead of '-'?

Comment (gabrielbarker) [reply qual_id/categories/games.py:13]: That's a good point. I'd prefer it if you remove it for now, but if do it in a single commit, then it'll be easy to revert in future if that's what we decide in future, thanks

Comment (mephi007) [reply qual_id/categories/games.py]: sure, I ll keep in mind. thanks

**Review by gabrielbarker — APPROVED** [review]
Thanks for getting these done so quickly

**Comment by gabrielbarker** [comment]
Great job and thanks a lot for the contribution @mephi007 !

**Comment by gabrielbarker** [comment]
The build is automated so the code should be live in a few minutes

**Comment by mephi007** [comment]
Thank you so much for the opportunity.. this was my first PR open-source... 
Hey @gabrielbarker is there anything as such end to end product, where I can contribute, as I know Java and Python.

**Comment by gabrielbarker** [comment]
Congratulations on your first PR!🎉🎉🎉
You did great, and you clearly have a good understanding of git.
There is not yet an end to end product for this project, just the API [here](https://qual-id.herokuapp.com/get/?pattern=animal-game) but I will let you know if that ever changes! Thanks a lot

**Pull request merged**